values.yaml

```diff
@@ -64,8 +64,8 @@ kpi:
       # -- KoBoCAT Bucket name 
       # KOBOCAT_AWS_STORAGE_BUCKET_NAME: ""
     secret:
-      # Overwrite these
-      DATABASE_URL: "postgres://postgres:password@postgres-postgresql:5432/postgres"
+      {}
+      # DATABASE_URL: "postgres://postgres:password@postgres-postgresql:5432/postgres"
       # KPI_BROKER_URL: "redis://localhost:6379/1"
       # CACHE_URL: "redis://localhost:6379/1"
```

Comment:
Same comment here for redis db indexes. (I cannot flag kobocat line 152, but it would be the same too).